src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,49 @@
+@import "../utils/variables";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-size;
+  padding: 10px;
+  gap: $gap-size;
+  margin: 0 auto;
+  font-size: 30px;
+  font-family: Arial, Helvetica, sans-serif;
+  cursor: pointer;
+
+  .calendar__day-box {
+    background-color: #eee;
+    width: $box-size;
+    height: $box-size;
+    border: 1px solid black;
+    box-sizing: border-box;
+    text-align: center;
+    line-height: $box-size;
+    transition: transform 0.5s;
```

Comment:
please add backgound-color to transition - you want to animate both, transform and background-color properties